code/game/objects/items/stacks/knife.dm

```diff
@@ -26,10 +25,63 @@
 	//spawn values
 	rarity_value = 8
 	spawn_tags = SPAWN_TAG_KNIFE
+	bad_type = /obj/item/stack/thrown
 
-/obj/item/stack/throwing_knife/update_icon()
+/obj/item/stack/thrown/update_icon()
 	icon_state = "[initial(icon_state)][amount]"
 
-/obj/item/stack/throwing_knife/examine(mob/user)
+/obj/item/stack/thrown/examine(mob/user)
 	if(..(user, 1))
 		to_chat(user, "There [src.amount == 1 ? "is" : "are"] [src.amount] [src.amount == 1 ? "knife" : "knives"] in the stack.")
```

Comment:
Would this say kinfe/knives even on the spears?

Reply:
Apparently so